types/react-router/index.d.ts

```diff
@@ -64,7 +64,7 @@ export interface RouteComponentProps<P> {
 
 export interface RouteProps {
   location?: H.Location;
-  component?: React.SFC<RouteComponentProps<any> | undefined> | React.ComponentClass<RouteComponentProps<any> | undefined>;
+  component?: React.SFC<RouteComponentProps<any> & any | undefined> | React.ComponentClass<RouteComponentProps<any> & any | undefined>;
```

Comment:
I thought it would have RouteComponentProps AND any thing else. Should I change to just any?